src/main/kotlin/com/github/mdashl/rankedhelper/commandhandler/Command.kt

```diff
@@ -5,19 +5,19 @@ import net.minecraft.command.ICommandSender
 import net.minecraftforge.client.ClientCommandHandler
 
 abstract class Command(private val alias: String) : CommandBase() {
-  override fun getCommandName(): String = alias
 
-  override fun getCommandUsage(sender: ICommandSender): String = "/$alias"
+    override fun getCommandName(): String = alias
 
-  override fun canCommandSenderUseCommand(sender: ICommandSender): Boolean = true
+    override fun getCommandUsage(sender: ICommandSender): String = "/$alias"
 
-  override fun processCommand(sender: ICommandSender, args: Array<out String>) {
-    execute(args.toList())
-  }
+    override fun canCommandSenderUseCommand(sender: ICommandSender): Boolean = true
 
-  abstract fun execute(args: List<String>)
+    override fun processCommand(sender: ICommandSender, args: Array<out String>): Unit = execute(args.toList())
```

Comment:
Don't use = when method returns Unit.
```suggestion
    override fun processCommand(sender: ICommandSender, args: Array<out String>) {
        execute(args.toList())
    }
```